wiki/Music_theory/Time_signature/de.md

```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+tags:
+  - sheet music
+  - Notenblatt
+---
+
+# Taktart
+
+*Für mehr Informationen über Taktarten in der Musiktheorie, siehe: [Taktangabe](https://de.wikipedia.org/wiki/Taktangabe)*
+
+Eine **Taktart** ist ein musikalischer Begriff, der die Anzahl an [Beats](/wiki/Music_theory/Beat) in einem einzigen [Takt](/wiki/Music_theory/Measure) eines Songs (die obere Zahl), und welche Note einen einzelnen Beat markiert (die untere Zahl), definiert. Bei einem 4/4-Takt (auch bekannt als "common time") passen 4 1/4 Noten (*Viertelnoten*) in einen Takt, wohingegen ein 3/4-Takt (auch bekannt als "Waltzer" oder "Dreivierteltakt") bedeutet, dass 3 1/4 Noten in einen Takt passen.
```

Comment:
Remove the `common time` remark - if there's no German name for it, just don't mention it

Reply:
```suggestion
Eine **Taktart** ist ein musikalischer Begriff, der die Anzahl an [Beats](/wiki/Music_theory/Beat) in einem einzigen [Takt](/wiki/Music_theory/Measure) eines Songs (die obere Zahl), und welche Note einen einzelnen Beat markiert (die untere Zahl), definiert. Bei einem 4/4-Takt passen 4 1/4-Noten (*Viertelnoten*) in einen Takt, wohingegen ein 3/4-Takt (auch bekannt als "Walzer" oder "Dreivierteltakt") bedeutet, dass 3 1/4-Noten in einen Takt passen.
```

Reply:
looks good